src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -355,8 +375,29 @@ public Visitable visit(ClassOrInterfaceDeclaration node, Void arg) {
           this.gotException = true;
         }
         addedTargetFiles.add(filePath);
+      } else {
+        UnsolvedClassOrInterface unsolvedInterface;
+        try {
+          implementedOrExtended.resolve();
+          continue;
+        } catch (UnsolvedSymbolException e) {
+          unsolvedInterface = new UnsolvedClassOrInterface(typeName, packageName);
+        }
+        // this extended/implemented type is an interface if it is in the declaration of an
+        // interface, or if it is used with the "implements" keyword.
+        boolean typeIsAnInterface =
+            node.isInterface() || implementedTypes.contains(implementedOrExtended);
+        if (typeIsAnInterface) {
+          unsolvedInterface.setIsAnInterface();
```

Comment:
This should be set during the constructor call (so that the field in `UnsolvedClassOrInterface` can be final).